etc/lime-elements.api.md

```diff
@@ -564,6 +564,8 @@ export namespace Components {
     export interface LimelProsemirrorAdapter {
         "contentType": 'markdown' | 'html';
         "language": Languages;
+        // Warning: (ae-incompatible-release-tags) The symbol "plugins" is marked as @beta, but its signature references "CustomElement" which is marked as @alpha
+        "plugins": CustomElement[];

```

Comment:
This warning means that we should mark the `plugins` prop as `@alpha`. I doesn't have an explicit `@beta` tag right now, but it, presumably, inherits the `@beta` tag from the class.

I'll see if adding `@alpha` to the prop removes this warning.

Reply:
#3263